next.config.js

```diff
@@ -120,7 +120,6 @@ module.exports = withSwingset({
         },
       ],
     },
-    outputFileTracing: false,
```

Comment:
This was turned off due to a bug, which was fixed in a v12 patch release shortly after we reported it.